src/_pytest/python.py

```diff
@@ -164,9 +164,10 @@ def async_warn_and_skip(nodeid: str) -> None:
     msg += (
         "You need to install a suitable plugin for your async framework, for example:\n"
     )
+    msg += "  - anyio\n"
     msg += "  - pytest-asyncio\n"
-    msg += "  - pytest-trio\n"
     msg += "  - pytest-tornasync\n"
+    msg += "  - pytest-trio\n"
     msg += "  - pytest-twisted"
```

Comment:
I ordered these alphabetically - maybe by github starts or pypi downloads would be better?

Reply:
Alphabetic is best for when looking at it later, for example, to check if a new one is already present or not.

Also no need to look up stars or downloads from elsewhere.

Reply:
Alphabetical seems good, even if it leaves 'me' at the end. `:]`